src/ape/api/accounts.py

```diff
@@ -137,6 +168,20 @@ def transfer(
         return self.call(txn, send_everything=value is None)
 
     def deploy(self, contract: ContractContainer, *args, **kwargs) -> ContractInstance:
+        """
+        Create a smart contract on the blockchain.
+
+        Method Limitations:
+            Smart Contract must compile before deploying.
+            A provider must be specified.
```

Comment:
Hmm... This line is a bit confusing, it makes it sound like they have to provide  a provider somehow.

Maybe say something like `A provider must be active.`